client/express-checkout/blocks/components/express-checkout-component.js

```diff
@@ -85,6 +85,18 @@ const ExpressCheckoutComponent = ( {
 		onReady( event );
 	};
 
+	if ( buttonOptions.buttonTheme.applePay === 'black' ) {
+		if ( expressPaymentMethod === 'applePay' ) {
+			buttonOptions.buttonHeight = buttonOptions.buttonHeight + 0.4;
+		}
+	} else if (
+		expressPaymentMethod === 'googlePay' &&
+		buttonOptions.buttonTheme.googlePay === 'white'
+	) {
+		// Adjust height for Google Pay only
+		buttonOptions.buttonHeight = buttonOptions.buttonHeight - 2;
+	}
```

Comment:
Nit: Minor suggestion to move this to a separate function similar to `getPaymentMethodsOverride`.

Reply:
done [f6805a6](https://github.com/Automattic/woocommerce-payments/pull/9258/commits/f6805a6734249f1a274f2c26e98c9fdbf9559f92)